textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/neextractor/NamedEntityExtractorTest.java

```diff
@@ -0,0 +1,174 @@
+package edu.uci.ics.textdb.dataflow.neextractor;
+
+import edu.uci.ics.textdb.api.common.ITuple;
+import edu.uci.ics.textdb.api.common.Schema;
+import edu.uci.ics.textdb.api.dataflow.ISourceOperator;
+
+import edu.uci.ics.textdb.api.storage.IDataReader;
+import edu.uci.ics.textdb.api.storage.IDataStore;
+import edu.uci.ics.textdb.api.storage.IDataWriter;
+import edu.uci.ics.textdb.common.constants.LuceneConstants;
+import edu.uci.ics.textdb.dataflow.neextrator.NamedEntityExtractor;
+import edu.uci.ics.textdb.dataflow.source.ScanBasedSourceOperator;
+import edu.uci.ics.textdb.dataflow.utils.TestUtils;
+import edu.uci.ics.textdb.storage.LuceneDataStore;
+import edu.uci.ics.textdb.storage.reader.LuceneDataReader;
+import edu.uci.ics.textdb.storage.writer.LuceneDataWriter;
+import org.apache.lucene.analysis.standard.StandardAnalyzer;
+import org.apache.lucene.queryparser.classic.QueryParser;
+import org.apache.lucene.search.Query;
+
+import org.apache.lucene.analysis.Analyzer;
+import org.junit.After;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author Feng [sam0227]
+ */
+public class NamedEntityExtractorTest {
+    private NamedEntityExtractor namedEntityExtractor;
+
+    private IDataWriter dataWriter;
+    private IDataReader dataReader;
+    private IDataStore dataStore;
+
+    private Query query;
+    private Analyzer analyzer;
+
+
+    @After
+    public void cleanUp() throws Exception {
```

Comment:
I don't have a strong opinion.  In general, I prefer uniformity.  @sam0227 please make a decision and move on.